releases/release-1.31/release-notes/maps/pr-124383-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 124383
+releasenote:
+  text: |-
+    kube-proxy's nftables mode (--proxy-mode=nftables) is now beta and available by default.
+
+    FIXME ADD MORE HERE BEFORE THE RELEASE, DOCS LINKS AND STUFF
```

Comment:
who needs to add this: FIXME ADD MORE HERE BEFORE THE RELEASE, DOCS LINKS AND STUFF

Reply:
We'll keep track of it [here](https://docs.google.com/spreadsheets/d/1ghhpEuLHsV6qdMhZrLjZ_pet7wOtN6_ESQDzAA8TcmA/edit?usp=sharing) and make sure the final release notes has this filled out!

Reply:
Marked it